dockerfiles/ci/alpine/php-8.0/Dockerfile

```diff
@@ -11,7 +11,7 @@ ARG phpTarGzUrl
 ARG phpSha256Hash
 RUN set -eux; \
     curl -fsSL -o /tmp/php.tar.gz "${phpTarGzUrl}"; \
-    (echo "${phpSha256Hash} /tmp/php.tar.gz" | sha256sum -); \
+    (echo "${phpSha256Hash} /tmp/php.tar.gz" | sha256sum -c -); \
```

Comment:
Good catch! Basically we weren't checking SHA on the development images?

Reply:
Correct. These would pass even when the hashes did not match.

Reply:
Again, good catch

Reply:
Thanks! :)